compiler/modules/front.syntax/ast.bal

```diff
@@ -459,6 +459,12 @@ public type GroupingTypeDesc record {|
     TypeDesc innerTd;
 |};
 
+public type OptionalTypeDesc record {|
```

Comment:
OptionalTypeDesc (and GroupingTypeDesc) are not consistent with UnaryTypeDesc below.

Reply:
fixed

Reply:
The field names need to be distinct.

This still isn't consistent: UnaryTypeDesc has an `op`.

For any change to the AST please propose a design in the issue before implementing.

Reply:
fixed